docs/file_explorer.md

```diff
@@ -1,76 +1,73 @@
-File Explorers
+File Explorer
 =============
 
-To get to the File Explorers module - click the file explorers icon.
+To get to the File Explorer module - click the File Explorer icon.
 
 ![](images/a-file-explorer1.png)
 
 File Storage
 ------------
-By default, Remix IDE stores files in **Workspaces** which are folders in your **browser's local storage**.  
+By default, Remix IDE stores files in **indexDB**.  
 
-**Important Note:** Clearing the browser storage will **permanently delete** all the files stored there. 
+Coding in Remix IDE Online is different from writing in a Google doc. Yes, both are written in a browser but a Google doc saves your work to Google’s servers, and Remix—out of the box—only saves your code to your browser’s storage. So tread carefully, browser storage is not permanent!
 
-If you want to use browser storage, but also to save a git repo on IPFS, use the **DGIT** plugin.
+**Important Note:** Clearing the browser storage & indexDB will **permanently delete** all the files stored there. 
 
-If you want to store files on your computer's filesystem, use **[Remixd](remixd.html)** or use the **[desktop version of Remix-IDE](https://github.com/ethereum/remix-desktop/releases/)**. Remixd enables you to have access to a selected folder on your hard drive. Remix Desktop is a version of Remix-IDE in an Electron app. 
+## File Storage outside of the browser and Version Control
 
-Workspaces
-------------
-Workspaces are new as of Remix IDE v0.11.0..  Workspaces should help clean up & organize your Files Explorer by allowing you to separate your projects.
+If you want to use browser storage, but also to save a git repo on IPFS, use the **DGIT** plugin.  
+
+If you want to store files on your computer's filesystem, use **[Remixd](remixd.html)** or use the **[desktop version of Remix-IDE](https://github.com/ethereum/remix-desktop/releases/)**. Remixd enables you to have access to a selected folder on your hard drive. Remix Desktop is a version of Remix-IDE in an Electron app. 
```

Comment:
I would suggest not to use `-` in `Remix-IDE`. We don't use it like that anymore